docs/moderator-onboarding-guide.md

```diff
@@ -0,0 +1,49 @@
+
+### Shadow a Mod
+
+All moderator actions can be followed by reviewing the [action logs](https://forum.freecodecamp.org/admin/logs/staff_action_logs). The actions taken by automated tools like 'akistmet' or 'system' can mostly be ignored until they result in a post that needs to be reviewed. Posts to be reviewed will show up in the [Review](https://forum.freecodecamp.org/review) area of the forum.
+
+For the first week or so you will want to pay attention to what is getting flagged and what is being reviewed and compare that to the actions being taken upon the flagged posts. So you may see the system account flag a post because the user created it too quickly. In many cases, the moderators will unflag the post by clicking on the "Approve Post" button or mark it as "Not Spam" (depending on the flag type).
+
+Commonly, spam flags can also be raised by members or moderators. Common duplicitious behaviour would involve opening an account, making a post that seems harmless, then editing that post later on to add a link to an external site for the purpose of advertising it. In this case, usually the member account is fairly new and has only made this one post thus far, which indicates that the account was opened for the sole purpose of spamming the community. The decision should be made to delete the account after the first offence in this case. The same applies for accounts whose first post is deemed to be spam.
+
+You may notice moderators performing a procedure called 'split topic'. Which can be interpreted as either a moderator has split a post that was made in error into the tail-end of a topic that is aleady closed into its own thread for eg.; or a moderator merged duplicate topics that a single user has created for the same fCC challenge. Watching this procedure will highlight different actions and their causes.
+
+Another useful feature that becomes enabled for all moderators is the ability to post a "Canned Reply" which is a pre-formed response that was worked out with the Mod team in order to quickly respond to some well-known and repetitive scenarios. These include:
+
+- welcoming a new forum member who has posted code without a question -> "Welcome - remind question"
+- reminding members not to post code solutions but to provide hints and tips instead -> "Solutions Instead of Help"
+- responding to a situation where someone's code works for you but not for them -> "Browser Issues"
+- encouraging members to open github issues when a possible bug is found -> "Bug Report"
+
+There are quite a few more which should either be self-explanatory or if not, they should be explained in a post in the Mod team subforum.
```

Comment:
I'd remove this line. Calling something "self-explanatory" can discourage someone when they *don't* understand it.

Reply:
How about:

There are quite a few more which you can read through to become familiar with their respective uses. Some of the history describing how these canned responses came to be or are in use may be found in the posts on the Mod-team subforum.

Reply:
```suggestion
There are more, which you can read through to become familiar with their respective uses. You can also find discussion around the templates in the mod-chat subforum, and you are welcome to ask questions if you aren't sure how a template should be used.
```

How's this?

Reply:
Pretty good but I would want to use the Mod-team name as that is the actual (somewhat unfortunate) name of the subforum.

There are many more that you can read through to become familiar with their respective uses. You can also find prior discussions around the templates in the Mod-chat subforum where you are welcome to ask further questions as needed.

Reply:
Whoops I messed up the name lol. Edited the suggestion.